apps/console/src/features/groups/components/wizard/group-basics.tsx

```diff
@@ -52,7 +51,7 @@ interface GroupBasicProps extends TestableComponentInterface {
 /**
  * Component to capture basic details of a new role.
  *
- * @param props Group Basic prop types
+ * @param props - props Group Basic prop types
```

Comment:
Isn't `props` redundant here?

```suggestion
 * @param props - Group Basic prop types
```

Reply:
Fixed with b3aa9c429a20e1316d5c35b3230e79785dfe75fe